iot_verifier/src/reward_share.rs

```diff
@@ -342,22 +341,29 @@ impl GatewayShares {
         let dc_transfer_rewards_per_share =
             rewards_per_share(total_dc_transfer_rewards_capped, total_dc_shares);
         // compute the awards per hotspot
+        tracing::info!("dc_transfer_rewards_per_share: {dc_transfer_rewards_per_share}");
         self.shares
             .into_iter()
-            .map(move |(hotspot_key, reward_shares)| proto::GatewayReward {
-                hotspot_key: hotspot_key.into(),
-                beacon_amount: compute_rewards(
-                    beacon_rewards_per_share,
-                    reward_shares.beacon_shares,
-                ),
-                witness_amount: compute_rewards(
-                    witness_rewards_per_share,
-                    reward_shares.witness_shares,
-                ),
-                dc_transfer_amount: compute_rewards(
-                    dc_transfer_rewards_per_share,
-                    reward_shares.dc_shares,
-                ),
+            .map(move |(hotspot_key, reward_shares)| {
+                tracing::info!(
```

Comment:
use the structured form please

```
tracing::info!(%hotspot_key, dc_shares = ?reward_shares.dc_shares, "dc shares for hotspot");
```

Note the shorthand `%` for using Display vs `?` for Debug. Also note the shorthand of not needing the = for hotspot_key if the name for the key you're outputting is what you want to see in the output

Reply:
this was added for debug purposes only, but noted